src/main/java/com/github/nkzawa/socketio/parser/Parser.java

```diff
@@ -57,10 +57,10 @@
         "CONNECT",
         "DISCONNECT",
         "EVENT",
-        "BINARY_EVENT",
         "ACK",
-        "BINARY_ACK",
         "ERROR",
+        "BINARY_EVENT",
+        "BINARY_ACK"
```

Comment:
This re ordering doesn't impact anything. This is just to follow the same index as the constant declared above in this file.